war/src/main/resources/images/symbols/shield-warning.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" class="ionicon" viewBox="0 0 512 512"><title>Shield Warning</title><path d="M250.26 165.39l5.74 122 5.73-121.95a5.74 5.74 0 00-5.79-6h0a5.74 5.74 0 00-5.68 5.95z" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32"/><path fill="currentColor" d="M256 397.25a20 20 0 1120-20 20 20 0 01-20 20z"/><path d="M463.1 112.37C373.68 96.33 336.71 84.45 256 48c-80.71 36.45-117.68 48.33-207.1 64.37C32.7 369.13 240.58 457.79 256 464c15.42-6.21 223.3-94.87 207.1-351.63z" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32"/></svg>
```

Comment:
The exclamation point looks a little far down to me.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" class="ionicon" viewBox="0 0 512 512"><title>Shield Warning</title><path d="M250.26 155.39l5.74 122 5.73-121.95a5.74 5.74 0 00-5.79-6h0a5.74 5.74 0 00-5.68 5.95z" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32"/><path fill="currentColor" d="M256 380.25a20 20 0 1120-20 20 20 0 01-20 20z"/><path d="M463.1 112.37C373.68 96.33 336.71 84.45 256 48c-80.71 36.45-117.68 48.33-207.1 64.37C32.7 369.13 240.58 457.79 256 464c15.42-6.21 223.3-94.87 207.1-351.63z" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32"/></svg>
```

WDYT? Before and after:

<img width="1107" alt="before" src="https://user-images.githubusercontent.com/1831569/190385368-482bbbca-cf99-466c-9459-182d16cb5c5f.png">
<img width="1107" alt="after" src="https://user-images.githubusercontent.com/1831569/190385446-f8e543a7-2bba-4683-bf5d-232de0281f9c.png">

Reply:
I agree, looks better 👍🏻 
Thanks :)